### DISCUSSION TITLE
Does Nuclei reporting module support not re-opening duplicate issues on GitHub?

### DISCUSSION THREAD

**Discussion by antin0de** [projectdiscovery/nuclei#4489] [Q&A]
Currently, if I use the GitHub integration for the reporting module, with `duplicate-issue-check: true`, it re-opens a closed issue when the same thing is detected again.

This is quite annoying as sometimes I want to close an issue as it is false-positive. Seems like the Jira integration supports `status-not: Closed`, do we have this support for GitHub?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
